### PR TITLE
Update .gitignore remove old inclusions and improve readability

### PR DESCRIPTION
**Pull request by iJustLeyxo**

### CODE REVIEW

**Comment by HrGaertner** [comment]
@MineMuse changes will then cause a merge conflict.

**Review by HrGaertner — APPROVED** [review]

**Review by HrGaertner — CHANGES_REQUESTED** [review]
Keep .vscode in gitignore

**Comment by iJustLeyxo** [comment]
Will do.

File: .gitignore [hunk 1]
```diff
@@ -1,39 +1,28 @@
 # General
-*
-!*/
-!*.txt
+  *
+  !*/
```
Comment (jon011235) [inline .gitignore:3]: What does this line do? Why would we want to include everything ending in a ','?

**Review by jon011235 — COMMENTED** [review]
Maybe also add *.c files?
@DM8AT already used them sometimes and if we want to have a software part that would be the starting point

What do you think?

File: .gitignore (hunk 1 above)
Comment (iJustLeyxo) [reply .gitignore:3]: I do not quite understand what you mean about files ending in "," being included. Could you please provide further information like line numbers please?

Have added c and asm file extensions.

**Review by jon011235 — APPROVED** [review]

**Pull request merged**